schemas/murmurations_map-v1.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://json-schema.org/draft-04/schema#",
+  "id": "https://cdn.murmurations.tech/schemas/murmurations_map-v1.json",
+  "title": "Murmurations Map",
+  "description": "Regenerative economy orgs - Select this schema to see your profile show up straight away on the Murmurations map",
+  "type": "object",
+  "properties": {
+    "linked_schemas": {
+      "$ref": "../fields/linked_schemas-v1.json"
+    },
+    "name": {
+      "$ref": "../fields/name-v1.json"
+    },
+    "url": {
+      "$ref": "../fields/urls-v1.json"
```

Comment:
@olisb Do you want to collect only one URL or enable the user to enter multiple URLs? If the former, use `url-v1.json`. If the latter, rename the properties to `urls`.

Reply:
URLs - done